drivah.toml.sh

```diff
@@ -6,17 +6,16 @@ VERSION=$(head $VERSION_FILE)
 
 DOCKER_REPO="docker.elastic.co"
 
-if [[ "$VERSION" == *-SNAPSHOT ]]; then
-  DOCKER_NAMESPACE="swiftype"
-else
-  DOCKER_NAMESPACE="enterprise-search"
-fi
+DOCKER_NAMESPACE="integrations"
 DOCKER_PROJECT_NAME="data-extraction-service"
 DOCKER_IMAGE_BASE="${DOCKER_REPO}/${DOCKER_NAMESPACE}/${DOCKER_PROJECT_NAME}"
 
+# We will continue to build and push image to namespace enterprise-search in 8.16 and 8.17
+TEMP_DOCKER_NAMESPACE="enterprise-search"
+TEMP_DOCKER_IMAGE_BASE="${DOCKER_REPO}/${TEMP_DOCKER_NAMESPACE}/${DOCKER_PROJECT_NAME}"
 
 cat <<EOF
 [container.image]
-names = ["${DOCKER_IMAGE_BASE}"]
+names = ["${DOCKER_IMAGE_BASE}", "${TEMP_DOCKER_IMAGE_BASE}"]
 tags = ["${VERSION}"]
```

Comment:
From the [drivah docs](https://drivah.elastic.dev/image_configuration_file.html#image-configuration-file):

>Multiple names and tags can be specified. Drivah will create the combinatorial product of all names with all tags.